docs/understanding-tracking-design/managing-your-tracking-scenarios/api/index.md

```diff
@@ -0,0 +1,300 @@
+- `scenario`: The definition of the tracking scenario, which should comply with the [validations](#validations).
+
+By default, the tracking scenario will be created with `scenario.status` set to `draft` and `scenario.version` set to `0` if not provided. These values can be changed and managed after creation. Here is an example response:
+
+```
+{
+    "data": [
+        {
+            "id": "5a203ef8-939b-4fd1-914e-f12a3dd1a869",
+            "version": 0,
+            "status": "draft",
+            "name": "Search",
+            "description": "Tracking the use of the search box",
+            "event": {
+                "source": "iglu:com.example/ui_actions/jsonschema/1-0-0"
+            }
+        }
+    ],
+    "includes": [
+        {
+            "author": "39b81015-1bd5-4b37-96c7-3296cabaa36f",
+            "message": "initial draft",
+            "date": "2023-04-26T14:41:48.708191Z",
+            "scenarioId": "5a203ef8-939b-4fd1-914e-f12a3dd1a869",
+            "version": 0,
+            "status": "draft",
+            "type": "History"
+        }
+    ],
+    "errors": []
+}
+```
+
+### Validations
+- `scenario.event.source`: If provided it should match a valid and existing Iglu URI.
+- `scenario.name`: It validates that the `scenario.name` of a tracking scenario is unique within the data structure context, inferred from the source data structure `scenario.event.source` if provided.
+- `scenario.version`: If provided should be equal or greater than zero.
+- `scenario.status`: If provided should match one of `draft`, `published` or `deprecated`.
+- `scenario.entities`: If provided it will validate that the entities, `scenario.entities.tracked` and `scenario.entities.enriched`, are not duplicated and that they exist.
+- `scneario.dataProductId`: If provided it will validate that the data products exists. (Coming soon)
+
+:::info
+This endpoint will trigger [**compatibility checking**](#compatibility-checks) if `event.source` and `event.schema` are defined.
+:::
+
+## Editing a Tracking Scenario
+
+Use this request to edit a tracking scenario within an organization. The format of the [request and response](#request-and-response) is the same format as during creation.
+
+`PUT /api/msc/v1/organizations/{organizationId}/tracking-scenarios/v1/{trackingScenarioId}`
+
+### Publishing a Tracking Scenario
+
+When editing a scenario, it can be published by setting the `status` to `published`. Currently, this will indicate to the tracking scenario consumers (for instance, front-end developers) that the tracking design is ready to be implemented or to be consumed.  
+By default, when a tracking scenario is created and no value is provided for `scenario.status`, it will be set to `draft`. With this, we suggests a tracking scenario lifecycle that we recommend following, but we allow certain degree of flexibility to accommodate unique customer use cases. Here is the suggested lifecycle: 
+
+```mermaid
+graph LR
```

Comment:
<img width="646" alt="image" src="https://github.com/snowplow/documentation/assets/2670454/e401c5ff-1ca2-43f4-b81a-47a801682233">

You might want to remove the fill from the lines ;)

Reply:
In fact I recommend minimizing the color changes so that we don’t break the dark mode

Reply:
I removed the style of the lines and that seems to have fixed it